lib/src/component/ref_util.dart

```diff
@@ -176,11 +177,11 @@ Ref<T> createRef<T>() {
 ///
 /// Learn more: <https://reactjs.org/docs/forwarding-refs.html>.
 UiFactory<TProps> Function(UiFactory<TProps>) forwardRef<TProps extends UiProps>(
```

Comment:
This should be deprecated and suggest using `uiForwardRef` instead

Reply:
🤦 Oh, you mean like it says in the AC? Sorry about that!

Reply:
No worries!